002/exercise/readme.md

```diff
@@ -6,9 +6,16 @@
 - Learn how to optimize a model through Gradient Descent
 
 # Task
-- Follow the Jupyter Notebook and complete the required tasks:
+- Follow the Jupyter Notebook  and complete the required tasks:
```

Comment:
This may be breaking some open source PR rules of engagement, but this additional space in not needed 
```suggestion
- Follow the Jupyter Notebook and complete the required tasks:
```

Reply:
Hey @pcuste1 , of course you are welcome to fix errors/typos. However, in this case, are you suggesting having double-space before the 'and'? Could you clarify and explain why?

Reply:
@gimseng , I think @pcuste1  is trying to say that the double-space before 'and' is not needed which seems to be there at the moment in this original commit.

Reply:
Oh my bet, thanks @aarpit1010, I think it didn't show up nicely on my phone. Thanks @pcuste1 !